x11/dmenu/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                dmenu
+version             4.7
+license             MIT/X
```

Comment:
The license seems to be just `MIT`.

Reply:
I looked here <http://git.suckless.org/dmenu/tree/LICENSE>. It think it's MIT/X, but I'm not a license expert. What do you think?

Reply:
According to Wikipedia,
> Because MIT has used many licenses for software, the Free Software Foundation considers "MIT License" ambiguous. "MIT License" may refer to the "Expat License" (used for Expat) or to the "X11 License" (also called "MIT/X Consortium License"; used for the X Window System by the MIT X Consortium).

MIT/X Consortium License should have `Except as contained in this notice, the name of the X Consortium shall not be used in advertising or otherwise to promote the sale, use or other dealings in this Software without prior written authorization from the X Consortium.` or `Except as contained in this notice, the name(s) of the above copyright holders shall not be used in advertising or otherwise to promote the sale, use or other dealings in this Software without prior written authorization.`.

Reply:
Also `MIT/X` is `X11` in our ports tree, see https://trac.macports.org/wiki/PortfileRecipes#licensekeyword.

Reply:
Gee - confusing enough if you're not a lawyer. So just MIT License then?

Reply:
IANAL, if you look into the actual license text it's `MIT`, if you only check the first line it's `X11`. We don't use `MIT/X` in all current Portfiles.

Reply:
Yep, you're right. It's the exact text of the MIT license. I'll change it to that.